.github/workflows/release.yaml

```diff
@@ -8,15 +8,20 @@ jobs:
     strategy:
       matrix:
         # Test in all supported versions of python in both Windows and Ubuntu.
-        python-version: ['3.11']
-        os: [ubuntu-latest]
+        python-version: ['3.9', '3.10', '3.11']
```

Comment:
Are we still testing 3.10? I thought we were keeping 3.9 and 3.11?

Reply:
On release, we are testing all the python versions for which we support. Same as with the csvcubed release.

Reply:
All python *minor* versions